include/swift/AST/Decl.h

```diff
@@ -6268,6 +6268,9 @@ class AbstractFunctionDecl : public GenericContext, public ValueDecl {
   /// Returns 'true' if the function is distributed.
   bool isDistributed() const;
 
+  /// Returns 'true' if the function can be back deployed.
```

Comment:
This comment doesn't really add any value in my opinion -- either it should be removed, or clarified with an explanation of the actual behavior.